tools/azure_iot_operations_stub_services/src/schema_registry.rs

```diff
@@ -180,7 +182,7 @@ impl From<service_gen::PutRequestSchema> for Schema {
                 .version
                 .expect("Schema version is required")
                 .parse()
-                .unwrap(), // TODO: Implement error handling for incorrect version number
-        }
+                .map_err(|_| "Invalid schema version, skipping request".to_string())?, // TODO: Implement error handling for incorrect version number
```

Comment:
should this be updated to just log, similar to the change you made in service.rs?

Reply:
The difference here is that I am doing a conversion from a PutRequest into a Schema, the only way to let the service.rs layer know that we have encountered an invalid version and skip that request is to surface the message as an error

When we receive a get we don't do a conversion so I can detect the invalid version in that layer and just skip the request